src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -464,13 +470,7 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
                         _ => None,
                     }
                 } else {
-                    // We already know this isn't in ValueNS, so no need to check variant_field
-                    return Err(ResolutionFailure::NotResolved {
-                        module_id,
-                        partial_res: Some(ty_res),
-                        unresolved: item_str.into(),
-                    }
-                    .into());
+                    None
```

Comment:
You dropped the comment, I think by accident.
```suggestion
                     // We already know this isn't in ValueNS, so no need to check variant_field
                    None
```

Reply:
No I also dropped the "no check" as well, it's handled down the line. I can add back the comment, but it will be funny that we still check if we aren't in ValueNS after that

Reply:
Ah ok, no need to add the comment back then.

Reply:
Wait I just realized this is not good like I left it.